src/main/kotlin/game/racingcar/strategy/RandomMoveStrategy.kt

```diff
@@ -5,12 +5,13 @@ import kotlin.random.Random
 class RandomMoveStrategy : MoveStrategy {
     override fun move(current: Int): Int {
         return when (Random.nextInt(0, 10)) {
-            in 4 until 10 -> current + MOVING_UNIT
+            in MOVABLE_RANGE -> current + MOVING_UNIT
             else -> current
         }
     }
 
     companion object {
         const val MOVING_UNIT = 1
+        val MOVABLE_RANGE = 4 until 10
```

Comment:
움직임 범위를 상수로 빼는 거 훌륭합니다! 👍👍

```suggestion
        private val MOVABLE_RANGE = 4..9
```
`until`를 사용하는 것보다 `IntRange`를 사용하여 코드의 가독성을 높이는 게 좋지 않을까요?